zerver/tests/test_subs.py

```diff
@@ -1627,6 +1627,19 @@ def test_subscriber_private_stream(self):
                                          invite_only=True)
         self.make_successful_subscriber_request(stream_name)
 
+    def test_json_get_subscribers(self):
+        # type: () -> None
+        """
+        json_get_subscribers also returns the list of subscribers for a stream.
+        """
+        stream_name = gather_subscriptions(self.user_profile)[0][0]['name']
+        result = self.client.post("/json/get_subscribers", {"stream": stream_name})
+        self.assert_json_success(result)
+        json = ujson.loads(result.content)
+
+        self.assertIn('subscribers', json)
+        self.assertIsInstance(json['subscribers'], list)
```

Comment:
Maybe worth checking this for a stream where we know the contents of the list, and confirming the values are correct?